net/uradvd/files/etc/init.d/uradvd

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh /etc/rc.common
+
+START=50
+
+USE_PROCD=1
+
+start_instance() {
+	local cfg="$1" enabled device ifname default_lifetime args
+
+	config_get enabled $cfg 'enabled'
+	config_get device $cfg 'device'
+	config_get ifname $cfg 'ifname'
+	config_get default_lifetime $cfg 'default_lifetime'
+
+	if [ "$enabled" != "1" ]; then
+		echo "disabled in config"
+		exit 0
+	fi
+
+	if [ -n "$device" ] && [ -n "$ifname" ]; then
+		echo "either set device or ifname"
```

Comment:
- Log messages should contain the instance name
- Errors and warnings should go to stderr (`>&2`)

Reply:
I agree.

Reply:
But the system log (`logread`) already contains the uradvd when it logs this output.